app/views/miq_request/_st_prov_show.html.haml

```diff
@@ -30,3 +30,9 @@
                             = render :partial => "miq_request/request_dialog_details",
                                      :locals => {:wf => wf, :field => field}
                       %hr
+
+  - record_ids = request_task_configuration_script_ids(@miq_request)
+  - if record_ids.size > 0
+    %h3
+      = _("Workflow States")
+    = react('RequestWorkflowStatus', {:ids => record_ids, :recordId => @miq_request.miq_request_tasks.first.options[:configuration_script_id]})
```

Comment:
I'm not sure why we need the `recordId` parameter, when you are passing them all in as ids?  Reason I mention is because you are getting the first one, which can't be accurate when you are expecting multiple.

Reply:
Talked through this with David, so I think we can just straight remove `recordId` because it's not actually used on the other side.